index.html

```diff
@@ -38,12 +38,16 @@
   <script type="riot/tag" src="tags/detail-app-container.tag"></script>
   <script type="riot/tag" src="tags/edit-app-container.tag"></script>
 
-  <script src="vendor/jquery-2.2.4.min.js"></script>
+  <script>
+    // https://github.com/electron/electron/issues/254
+    window.$ = window.jQuery = require('./vendor/jquery-2.2.4.min.js');
+  </script>
   <script src="vendor/bootstrap-3.3.6-dist/js/bootstrap.min.js"></script>
   <script src="vendor/riot/riot+compiler.min.js"></script>
-  <script src="scripts/indexApi.js"></script>
+  <!-- <script src="scripts/indexApi.js"></script> -->
 
   <script>
+  var Apps = require('./lib/api')
```

Comment:
Here, we can't use ''script/indexApi.js' inside of  'require('./lib/api')' => Using './lib/api' is more recommended? ;)